src/test/java/com/google/cloud/resourcemanager/ResourceManagerImplTest.java

```diff
@@ -454,4 +489,226 @@ public void testRuntimeException() {
     thrown.expectMessage(exceptionMessage);
     resourceManagerMock.get(PARTIAL_PROJECT.getProjectId());
   }
+
+  @Test
+  public void testClearOrgPolicy() {
+    try {
+      ClearOrgPolicyRequest requestBody = new ClearOrgPolicyRequest();
+      requestBody.setConstraint(CONSTRAINTS);
+      requestBody.setEtag(ETAG);
+      RESOURCE_MANAGER.clearOrgPolicy(RESOURCE, requestBody);
+      fail("Should fail because the organization policy doesn't exist.");
+    } catch (ResourceManagerException e) {
```

Comment:
e --> expected in cases like this where we want the exception to be thrown

Reply:
done